modules/ossm-deploy-application-workloads-per-user-group.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+// install/ossm-using-multiple-control-planes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-per-user-group_{context}"]
+= Deploy application workloads per user group
+:context: ossm-deploy-application-workloads-per-user-group
+
+This procedure demonstrates how to deploy application workloads in separate namespaces based on user groups.
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-1
+----
+
+. Label the namespace with the respective control plane:
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 usergroup=usergroup-1 istio.io/rev=mesh-1
+----
+
+. Deploy the `sleep` and `httpbin` applications:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml
+$ oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml
+----
+
+. Wait for the `httpbin` and `sleep` pods to run with sidecars injected:
++
+[source,terminal]
+----
+$ oc get pods -n app-ns-1
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-7f56dc944b-kpw2x   2/2     Running   0          2m26s
+sleep-5577c64d7c-b5wd2     2/2     Running   0          91m
+----
+
+. Repeat the above steps for two more namespaces: `app-ns-2` and `app-ns-3`, adding them both to the `usergroup-2` revision.
+
+[NOTE]
+====
+You can use the following command to wait for deployment to be ready:
+[source,terminal]
+----
+$ oc wait deployment/sleep -n app-ns-1 --for=condition=Available
+----
+====
+
+.Verification
+
+After deploying the applications, use the `istioctl ps` command to verify that each application workload is managed by its assigned control plane. For example, `app-ns-1` is managed by `mesh-1`, and `app-ns-2` and `app-ns-3` are managed by `mesh-2`.
```

Comment:
I found this step really useful, but also not easy to understand, because "istioctl ps" on its own doesn't work, and you need to specify the namespace of the control plane, which isn't given in this paragraph. It could also use example output. 

I suggest the following content replace this paragraph (@shreyasiddhartha feel free to adjust the formatting):

----
After deploying the applications, use the `istioctl ps` command to verify that each application workload is managed by its assigned control plane. 

For the first control plane deployed in `istio-system-1`:
```
istioctl ps -i istio-system-1
NAME                                  CLUSTER        CDS              LDS              EDS              RDS              ECDS        ISTIOD                            VERSION
httpbin-7f56dc944b-vwfm5.app-ns-1     Kubernetes     SYNCED (11m)     SYNCED (11m)     SYNCED (11m)     SYNCED (11m)     IGNORED     istiod-mesh-1-b69646b6f-kxrwk     1.23.0
sleep-5577c64d7c-d675f.app-ns-1       Kubernetes     SYNCED (11m)     SYNCED (11m)     SYNCED (11m)     SYNCED (11m)     IGNORED     istiod-mesh-1-b69646b6f-kxrwk     1.23.0
```
For the second control plane deployed in `istio-system-2`:
```
istioctl ps -i istio-system-2
NAME                                  CLUSTER        CDS                LDS                EDS                RDS                ECDS        ISTIOD                            VERSION
httpbin-7f56dc944b-54gjs.app-ns-3     Kubernetes     SYNCED (3m59s)     SYNCED (3m59s)     SYNCED (3m59s)     SYNCED (3m59s)     IGNORED     istiod-mesh-2-8666fdfc6-mqp45     1.23.0
httpbin-7f56dc944b-gnh72.app-ns-2     Kubernetes     SYNCED (4m1s)      SYNCED (4m1s)      SYNCED (3m59s)     SYNCED (4m1s)      IGNORED     istiod-mesh-2-8666fdfc6-mqp45     1.23.0
sleep-5577c64d7c-k9mxz.app-ns-2       Kubernetes     SYNCED (4m1s)      SYNCED (4m1s)      SYNCED (3m59s)     SYNCED (4m1s)      IGNORED     istiod-mesh-2-8666fdfc6-mqp45     1.23.0
sleep-5577c64d7c-m9hvm.app-ns-3       Kubernetes     SYNCED (4m1s)      SYNCED (4m1s)      SYNCED (3m59s)     SYNCED (4m1s)      IGNORED     istiod-mesh-2-8666fdfc6-mqp45     1.23.0
```